app/models/valid_fulfilled_at_date.rb

```diff
@@ -0,0 +1,56 @@
+class ValidFulfilledAtDate
+
+  include ActiveModel::Validations
+  include DateHelper
+
+  validate :valid_format
+  validate :not_in_future, if: :to_time
+  validate :recent, if: :to_time
+
+  def self.min
+    # Beginning of previous fiscal year
```

Comment:
It's what the client asked for. They're mainly concerned about is the changeover to the new fiscal year as they're closing out the previous year. They want to be able to place orders into the previous fiscal year.